controllers/periodic_healthcheck_test.go

```diff
@@ -130,6 +130,7 @@ func TestStartHealthCheckLoopWithCustomRetries(t *testing.T) {
 
 	mockRt.EXPECT().RoundTrip(gomock.Any()).Return(nil, errors.New("error")).Times(9)
 	mockEtcd.EXPECT().MemberList(gomock.Any()).Return(etcdTest.getMemberListResponse(), nil).Times(3)
+	mockEtcd.EXPECT().MemberRemove(gomock.Any(), gomock.Any()).Return(etcdTest.getMemberRemoveResponse(), nil).Times(1)
```

Comment:
curious why is this test changing?

Reply:
I updated the memberlist response to append port 2380 instead of 2379 to the addresses, as this is what `etcdMemberRemove` expects to be the peer URL for the member being removed. Now the addresses matched, so member remove is actually called and should be expected in the test